test/routes/actions.test.js

```diff
@@ -1,7 +1,7 @@
 const ApplicationContext = require('../../src/context/application-context');
 const ActionsRoutes = require('../../src/routes/actions');
 
-function initContext(schema, smartActionHookGetResponse) {
```

Comment:
This is cosmetic change not related to or right?

Reply:
yes cosmetic :)